css/main.css

```diff
@@ -3,18 +3,26 @@
 	--base-color: #E5D9B9;
 	--lighter-color: #FBEFD5;
 	--lightest-color: #FFF5B5;
-	--topbar-height: 2.5em;
 }
 
 html{
```

Comment:
I'm pretty sure we don't actually need any styles in the html tag, it's set to take up 100% of width and height by default, and by removing our styles it doesn't seem to change anything.

![image](https://user-images.githubusercontent.com/47337746/215372248-ff997d0b-c810-42e9-b1f8-5d0c3c9ceed1.png)

Reply:
It looks like you are right. Fixed.